inference-engine/src/inference_engine/src/cpp_interfaces/interface/ie_iplugin_internal.cpp

```diff
@@ -219,6 +225,85 @@ void IInferencePlugin::SetExeNetworkInfo(const std::shared_ptr<IExecutableNetwor
     // Set inputs/outputs and pointer to plugin manually here
     exeNetwork->setNetworkInputs(copyInfo(constMapCast(inputs)));
     exeNetwork->setNetworkOutputs(copyInfo(constMapCast(outputs)));
+
+    ngraph::ParameterVector parameters;
+    ngraph::ResultVector results;
+    std::vector<ngraph::Output<ngraph::Node>> node_outputs;
+
+    for (auto&& input : inputs) {
+        auto tensor_desc = input.second->getTensorDesc();
+        auto dims = tensor_desc.getDims();
+        parameters.push_back(std::make_shared<ngraph::op::v0::Parameter>(
+            details::convertPrecision(tensor_desc.getPrecision()),
+            ngraph::PartialShape{std::vector<ngraph::Dimension>{dims.begin(), dims.end()}}));
+        parameters.back()->set_friendly_name(input.first);
+        node_outputs.push_back(parameters.back()->output(0));
+    }
+
+    auto node = std::make_shared<ExecGraphInfoSerialization::ExecutionNode>(node_outputs, outputs.size());
+
+    int i = 0;
+    for (auto&& output : outputs) {
+        auto tensor_desc = output.second->getTensorDesc();
+        auto dims = tensor_desc.getDims();
+        node->set_output_type(i,
+                              details::convertPrecision(tensor_desc.getPrecision()),
+                              {std::vector<ngraph::Dimension>{dims.begin(), dims.end()}});
+        results.push_back(std::make_shared<ngraph::op::v0::Result>(node->output(i)));
+        ++i;
+    }
+    exeNetwork->setRuntimeFunction(std::make_shared<ov::Function>(results, parameters, "execution_info"));
```

Comment:
do we guarantee that order of results / parameters here is the same as in original ngraph function?

Reply:
no. 
This pass is used for several test cases in GNU plugin that use deprecated IRv6 network without ngraph implementation.
For ngraph implementation, the order of parameters will be preserved.